src/main/integrations/onuncaughtexception.ts

```diff
@@ -37,8 +38,16 @@ export class OnUncaughtException implements Integration {
             level: Severity.Fatal,
           }));
 
+          let theError = error;
+          if (!isError(error) && error.message && error.stack && error.name) {
```

Comment:
Note that this is falsy if any of the attributes is empty. Do you rather mean to check for `undefined`?

Reply:
Good point, I made it to only check for `stack` which makes it less strict.

Reply:
Sounds good!